bili.config.js

```diff
@@ -30,5 +32,8 @@ module.exports = {
       sourceMap: true,
       open: false
     }
+  },
+  resolvePlugins: {
+    alias: require('@rollup/plugin-alias')
```

Comment:
Once https://github.com/egoist/bili/pull/259 gets merged we can get rid of this ugliness 🙏